Plugins/ExaTrkX/include/Acts/Plugins/ExaTrkX/detail/buildEdges.hpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <cstdint>
+
+namespace at {
+class Tensor;
+}
+
+namespace Acts {
+namespace detail {
+
+/// Post process edges
+at::Tensor postprocessEdgeTensor(at::Tensor edges, bool removeSelfLoops = true,
```

Comment:
`edges` by value on purpose? or is this more like a pointer type?

Reply:
`at::Tensor` is reference counted like `std::shared_ptr`, so I would keep it like this for now.